qiskit_nature/second_q/problems/electronic_structure_problem.py

```diff
@@ -312,11 +312,32 @@ def symmetry_sector_locator(
         return sector
 
     @staticmethod
-    def _pick_sector(z2_symmetries: Z2Symmetries, hf_str: List[bool]) -> List[int]:
+    def _pick_sector(
+        z2_symmetries: Z2Symmetries | Z2SparseSymmetries, hf_str: List[bool]
+    ) -> List[int]:
         # Finding all the symmetries using the find_Z2_symmetries:
         taper_coeff: List[int] = []
         for sym in z2_symmetries.symmetries:
             coeff = -1 if np.logical_xor.reduce(np.logical_and(sym.z, hf_str)) else 1
             taper_coeff.append(coeff)
 
         return taper_coeff
+
+    def get_tapered_mapper(self, mapper: QubitMapper) -> TaperedQubitMapper:
```

Comment:
It should be possible to define this directly on `BaseProblem`, right?

Reply:
Done